consumer/README.md

```diff
@@ -0,0 +1,64 @@
+## How to run consumer chain
```

Comment:
Who is the target audience for this readme and the included shell scripts?Considering this repo only has example binaries and won't be used in production. Is this info for testing the sovereign->consumer transition in a testnet?

Reply:
This is for local testing of sovereign to consumer transition for chain developers.

Reply:
Sounds good, these files should probably be moved to a directory like `tests` then, to make it clear this is all for local testing

Reply:
Okay will move this to tests directory

Reply:
Moved to `tests/sovereign_consumer_upgrade_local`